go/pkg/tools/tools.go

```diff
@@ -18,8 +18,8 @@ import (
 	"os"
 
 	"github.com/golang/protobuf/proto"
```

Comment:
This should be`google.golang.org/protobuf/proto`. There are several other places where the new apiv2 protobuf bindings look to be used but the protobuf package references is the old apiv1 package